2021-09-14 10:37:05,064 INFO    MainThread:1390444 [internal.py:wandb_internal():91] W&B internal server running at pid: 1390444, started at: 2021-09-14 10:37:05.064155
2021-09-14 10:37:05,066 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: check_version
2021-09-14 10:37:05,066 INFO    WriterThread:1390444 [datastore.py:open_for_write():80] open: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/run-ze900pqk.wandb
2021-09-14 10:37:05,068 DEBUG   SenderThread:1390444 [sender.py:send():179] send: header
2021-09-14 10:37:05,068 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: check_version
2021-09-14 10:37:05,099 DEBUG   SenderThread:1390444 [sender.py:send():179] send: run
2021-09-14 10:37:05,162 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:05,162 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:06,269 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:06,269 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:08,509 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:08,509 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:08,509 ERROR   SenderThread:1390444 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 133, in execute
    six.reraise(*sys.exc_info())
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 719, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 127, in execute
    return self.client.execute(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 39, in execute
    request.raise_for_status()
  File "/usr/local/lib/python3.6/dist-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql
2021-09-14 10:37:13,556 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:13,557 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:23,501 INFO    SenderThread:1390444 [dir_watcher.py:__init__():168] watching files in: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files
2021-09-14 10:37:23,501 INFO    SenderThread:1390444 [sender.py:_start_run_threads():719] run started: ze900pqk with start time 1631630224
2021-09-14 10:37:23,501 DEBUG   SenderThread:1390444 [sender.py:send():179] send: summary
2021-09-14 10:37:23,502 INFO    SenderThread:1390444 [sender.py:_save_file():841] saving file wandb-summary.json with policy end
2021-09-14 10:37:23,503 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: run_start
2021-09-14 10:37:24,503 INFO    Thread-8  :1390444 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/wandb-summary.json
2021-09-14 10:37:24,582 DEBUG   HandlerThread:1390444 [meta.py:__init__():39] meta init
2021-09-14 10:37:24,582 DEBUG   HandlerThread:1390444 [meta.py:__init__():53] meta init done
2021-09-14 10:37:24,582 DEBUG   HandlerThread:1390444 [meta.py:probe():210] probe
2021-09-14 10:37:24,588 DEBUG   HandlerThread:1390444 [meta.py:_save_pip():57] save pip
2021-09-14 10:37:24,589 DEBUG   HandlerThread:1390444 [meta.py:_save_pip():71] save pip done
2021-09-14 10:37:24,589 DEBUG   HandlerThread:1390444 [meta.py:probe():252] probe done
2021-09-14 10:37:24,591 DEBUG   SenderThread:1390444 [sender.py:send():179] send: files
2021-09-14 10:37:24,591 INFO    SenderThread:1390444 [sender.py:_save_file():841] saving file wandb-metadata.json with policy now
2021-09-14 10:37:24,597 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: stop_status
2021-09-14 10:37:24,597 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: stop_status
2021-09-14 10:37:24,650 ERROR   Thread-11 :1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:24,651 ERROR   Thread-11 :1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:24,656 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:24,657 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:25,504 INFO    Thread-8  :1390444 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/output.log
2021-09-14 10:37:25,505 INFO    Thread-8  :1390444 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/wandb-metadata.json
2021-09-14 10:37:25,505 INFO    Thread-8  :1390444 [dir_watcher.py:_on_file_created():216] file/dir created: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/requirements.txt
2021-09-14 10:37:25,908 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:25,909 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:25,909 ERROR   SenderThread:1390444 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 133, in execute
    six.reraise(*sys.exc_info())
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 719, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 127, in execute
    return self.client.execute(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 39, in execute
    request.raise_for_status()
  File "/usr/local/lib/python3.6/dist-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql
2021-09-14 10:37:25,952 ERROR   Thread-11 :1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:25,952 ERROR   Thread-11 :1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:25,953 ERROR   Thread-11 :1390444 [retry.py:__call__():126] Retry attempt failed:
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/lib/retry.py", line 102, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 133, in execute
    six.reraise(*sys.exc_info())
  File "/usr/local/lib/python3.6/dist-packages/six.py", line 719, in reraise
    raise value
  File "/usr/local/lib/python3.6/dist-packages/wandb/sdk/internal/internal_api.py", line 127, in execute
    return self.client.execute(*args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
  File "/usr/local/lib/python3.6/dist-packages/wandb/vendor/gql-0.2.0/gql/transport/requests.py", line 39, in execute
    request.raise_for_status()
  File "/usr/local/lib/python3.6/dist-packages/requests/models.py", line 943, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 429 Client Error: Too Many Requests for url: https://api.wandb.ai/graphql
2021-09-14 10:37:28,134 ERROR   Thread-11 :1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:28,134 ERROR   Thread-11 :1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:28,145 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:28,145 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:33,259 INFO    Thread-11 :1390444 [upload_job.py:push():137] Uploaded file /tmp/3287117.1.gpu_ded.q/tmpa2itnqcqwandb/4x6aa46e-wandb-metadata.json
2021-09-14 10:37:48,022 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: stop_status
2021-09-14 10:37:48,022 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: stop_status
2021-09-14 10:37:52,946 DEBUG   SenderThread:1390444 [sender.py:send():179] send: stats
2021-09-14 10:37:56,954 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:37:56,954 DEBUG   SenderThread:1390444 [sender.py:send():179] send: telemetry
2021-09-14 10:37:56,954 DEBUG   SenderThread:1390444 [sender.py:send():179] send: exit
2021-09-14 10:37:56,954 INFO    SenderThread:1390444 [sender.py:send_exit():287] handling exit code: 1
2021-09-14 10:37:56,955 INFO    SenderThread:1390444 [sender.py:send_exit():295] send defer
2021-09-14 10:37:56,955 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:37:56,955 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:37:56,955 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 0
2021-09-14 10:37:56,955 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:37:56,955 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 0
2021-09-14 10:37:56,955 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 1
2021-09-14 10:37:56,955 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:37:56,955 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 1
2021-09-14 10:37:56,989 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:37:56,990 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 1
2021-09-14 10:37:56,990 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 2
2021-09-14 10:37:56,990 DEBUG   SenderThread:1390444 [sender.py:send():179] send: stats
2021-09-14 10:37:56,990 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:37:56,992 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 2
2021-09-14 10:37:56,992 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:37:56,993 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 2
2021-09-14 10:37:56,993 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 3
2021-09-14 10:37:56,993 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:37:56,993 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 3
2021-09-14 10:37:56,993 DEBUG   SenderThread:1390444 [sender.py:send():179] send: summary
2021-09-14 10:37:56,994 INFO    SenderThread:1390444 [sender.py:_save_file():841] saving file wandb-summary.json with policy end
2021-09-14 10:37:56,994 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:37:56,994 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 3
2021-09-14 10:37:56,994 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 4
2021-09-14 10:37:56,994 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:37:56,994 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 4
2021-09-14 10:37:56,994 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:37:56,994 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 4
2021-09-14 10:37:57,049 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:57,050 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:37:57,056 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:37:57,517 INFO    Thread-8  :1390444 [dir_watcher.py:_on_file_modified():229] file/dir modified: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/wandb-summary.json
2021-09-14 10:37:58,210 ERROR   SenderThread:1390444 [internal_api.py:execute():130] 429 response executing GraphQL.
2021-09-14 10:37:58,210 ERROR   SenderThread:1390444 [internal_api.py:execute():131] {"error":"rate limit exceeded"}

2021-09-14 10:38:00,705 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 5
2021-09-14 10:38:00,705 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:00,706 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:38:00,706 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 5
2021-09-14 10:38:00,706 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:38:00,706 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 5
2021-09-14 10:38:00,706 INFO    SenderThread:1390444 [dir_watcher.py:finish():282] shutting down directory watcher
2021-09-14 10:38:00,810 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:01,518 INFO    SenderThread:1390444 [dir_watcher.py:_on_file_modified():229] file/dir modified: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/config.yaml
2021-09-14 10:38:01,519 INFO    SenderThread:1390444 [dir_watcher.py:finish():312] scan: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files
2021-09-14 10:38:01,519 INFO    SenderThread:1390444 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/wandb-metadata.json wandb-metadata.json
2021-09-14 10:38:01,519 INFO    SenderThread:1390444 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/config.yaml config.yaml
2021-09-14 10:38:01,519 INFO    SenderThread:1390444 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/requirements.txt requirements.txt
2021-09-14 10:38:01,519 INFO    SenderThread:1390444 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/wandb-summary.json wandb-summary.json
2021-09-14 10:38:01,519 INFO    SenderThread:1390444 [dir_watcher.py:finish():318] scan save: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/output.log output.log
2021-09-14 10:38:01,520 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 6
2021-09-14 10:38:01,520 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:01,526 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:38:01,526 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 6
2021-09-14 10:38:01,531 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:38:01,531 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 6
2021-09-14 10:38:01,532 INFO    SenderThread:1390444 [file_pusher.py:finish():177] shutting down file pusher
2021-09-14 10:38:01,631 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:01,631 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:01,733 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:01,733 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:01,835 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:01,836 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:01,899 INFO    Thread-13 :1390444 [upload_job.py:push():137] Uploaded file /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/requirements.txt
2021-09-14 10:38:01,900 INFO    Thread-14 :1390444 [upload_job.py:push():137] Uploaded file /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/wandb-summary.json
2021-09-14 10:38:01,919 INFO    Thread-12 :1390444 [upload_job.py:push():137] Uploaded file /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/files/config.yaml
2021-09-14 10:38:01,938 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:01,939 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:02,040 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:02,040 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:02,120 INFO    Thread-7  :1390444 [sender.py:transition_state():308] send defer: 7
2021-09-14 10:38:02,120 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:38:02,121 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 7
2021-09-14 10:38:02,121 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:38:02,121 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 7
2021-09-14 10:38:02,142 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:02,179 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 8
2021-09-14 10:38:02,179 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:02,179 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:38:02,180 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 8
2021-09-14 10:38:02,180 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:38:02,180 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 8
2021-09-14 10:38:02,180 INFO    SenderThread:1390444 [sender.py:transition_state():308] send defer: 9
2021-09-14 10:38:02,180 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: defer
2021-09-14 10:38:02,181 DEBUG   SenderThread:1390444 [sender.py:send():179] send: final
2021-09-14 10:38:02,181 INFO    HandlerThread:1390444 [handler.py:handle_request_defer():141] handle defer: 9
2021-09-14 10:38:02,181 DEBUG   SenderThread:1390444 [sender.py:send():179] send: footer
2021-09-14 10:38:02,181 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: defer
2021-09-14 10:38:02,181 INFO    SenderThread:1390444 [sender.py:send_request_defer():304] handle sender defer: 9
2021-09-14 10:38:02,284 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: poll_exit
2021-09-14 10:38:02,284 DEBUG   SenderThread:1390444 [sender.py:send_request():193] send_request: poll_exit
2021-09-14 10:38:02,284 INFO    SenderThread:1390444 [file_pusher.py:join():182] waiting for file pusher
2021-09-14 10:38:02,286 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: get_summary
2021-09-14 10:38:02,286 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: sampled_history
2021-09-14 10:38:02,286 DEBUG   HandlerThread:1390444 [handler.py:handle_request():124] handle_request: shutdown
2021-09-14 10:38:02,286 INFO    HandlerThread:1390444 [handler.py:finish():638] shutting down handler
2021-09-14 10:38:03,181 INFO    WriterThread:1390444 [datastore.py:close():288] close: /grid/koo/home/toneyan/profile/QuantPred/wandb/run-20210914_103704-ze900pqk/run-ze900pqk.wandb
2021-09-14 10:38:03,285 INFO    SenderThread:1390444 [sender.py:finish():945] shutting down sender
2021-09-14 10:38:03,285 INFO    SenderThread:1390444 [file_pusher.py:finish():177] shutting down file pusher
2021-09-14 10:38:03,285 INFO    SenderThread:1390444 [file_pusher.py:join():182] waiting for file pusher
2021-09-14 10:38:03,287 INFO    MainThread:1390444 [internal.py:handle_exit():78] Internal process exited
